meetings/README.md

```diff
@@ -31,7 +31,7 @@ You can add this full list of meetings to your personal calendar by importing by
   ([ical](https://raw.githubusercontent.com/ansible/community/main/meetings/ical/azure.ics))
   in `#ansible-azure`
 
-* [15:00 UTC](http://www.thetimezoneconverter.com/?t=15:00&tz=UTC):
+* [16:00 UTC](http://www.thetimezoneconverter.com/?t=16:00&tz=UTC):
```

Comment:
@samccann is this being adjusted per the US DST? If yes, it'd be easier to set time in one of the local US timezones instead of UTC, in the calendar files. This way it'd happen automatically, w/o having to change the files twice a yeah.

Reply:
I don't think it's possible, unfortunately. It looks like these calendars are set up as yaml files run through [yaml2ical](https://pypi.org/project/yaml2ical/) and near as I can tell, that requires UTC. @gundalow do you know if there's a way to do this w/o having to change the calendar twice a year?